spec/acceptance/steps/links_indexing_steps.rb

```diff
@@ -0,0 +1,114 @@
+step 'I create a draft content item' do
+  @content_id = SecureRandom.uuid
+  @base_path = "/" + SecureRandom.hex
+
+  payload = {
+    title: 'Hallo',
+    base_path: @base_path,
+    schema_name: 'detailed_guide',
+    document_type: 'detailed_guide',
+    publishing_app: 'end-to-end-publishing-test',
+    rendering_app: 'end-to-end-publishing-test',
+    locale: 'en',
+    routes: [
+      { path: @base_path, type: 'exact' }
+    ]
+  }
+
+  Services.publishing_api.put_content(@content_id, payload)
+
+  sleep 0.1
```

Comment:
Are the sleeps to account for the Sidekiq queue? It'd be worth doing the work up front to do something similar to Capybara's `wait_until` syntax, else this suite will become quite brittle.